### ISSUE TITLE
Dart roller stopped on deprecation analyzer error

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#123157]
Dart roll reverted in https://github.com/flutter/engine/pull/40496

```
info • 'ByteConversionSinkBase' is deprecated and shouldn't be used. Use ByteConversionSink instead • packages/flutter/lib/src/foundation/consolidate_response.dart:101:29 • deprecated_member_use
```

in https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/foundation/consolidate_response.dart#L101

Symbol deprecated in https://dart.googlesource.com/sdk.git/+/b2f4cf3e01edf2a26a031ab7151c282713c8bd1b cc @lrhn 

cc @aam

**Comment by aam** [comment]
Revert landed, 3.0.0-355.0.dev should have the revert, started the roll of that version into the engine

**Comment by aam** [comment]
Turns out the revert didn't make it into 355, but is part of 356
